<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeniorWindow</class>
 <widget class="QWidget" name="SeniorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeniorWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="ideasLabel">
     <property name="text">
      <string>Ideas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="ideasList">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="actLabel">
       <property name="text">
        <string>Act:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="actLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="addIdeaButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Add Idea</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="removeIdeaButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Remove Idea</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="savePlotButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Save Plot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="acceptButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addIdeaButton</sender>
   <signal>clicked()</signal>
   <receiver>SeniorWindow</receiver>
   <slot>addIdeaCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeIdeaButton</sender>
   <signal>clicked()</signal>
   <receiver>SeniorWindow</receiver>
   <slot>removeIdeaCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savePlotButton</sender>
   <signal>clicked()</signal>
   <receiver>SeniorWindow</receiver>
   <slot>savePlotCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>SeniorWindow</receiver>
   <slot>acceptCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ideasList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>SeniorWindow</receiver>
   <slot>checkStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>-167</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addIdeaCommand()</slot>
  <slot>savePlotCommand()</slot>
  <slot>removeIdeaCommand()</slot>
  <slot>acceptCommand()</slot>
  <slot>checkStatus()</slot>
 </slots>
</ui>
